tests/FakeItEasy.Specs/XBehaveExtensions.cs

```diff
@@ -0,0 +1,70 @@
+namespace FakeItEasy.Specs
+{
+    using System;
+    using System.Diagnostics.CodeAnalysis;
+    using Creation;
+    using Xbehave;
+    using Xbehave.Sdk;
+
+    public static class XBehaveExtensions

```

Comment:
I think (correct me if I'm wrong) we usually prefer `{Type}Extensions`, which would make this `StringExtensions`.